app/views/layouts/megatron/application.html.slim

```diff
@@ -8,10 +8,12 @@ html lang="en"
     meta name="viewport" content="width=device-width,initial-scale=1"
     meta charset="utf-8"
     meta http-equiv="X-UA-Compatible" content="IE=edge"
+    meta name="bugsnag" data-api-key="#{ENV['BUGSNAG_API_KEY']}" data-release-stage="#{Rails.env}"
```

Comment:
Yes, all of our "staging" apps are `Rails.env.production?`. That way their configuration is closer to production 😆